Content.Shared/Stacks/SharedStackComponent.cs

```diff
@@ -9,7 +9,7 @@ public abstract class SharedStackComponent : Component
     {
         [ViewVariables(VVAccess.ReadWrite)]
         [DataField("stackType", required:true, customTypeSerializer:typeof(PrototypeIdSerializer<StackPrototype>))]
-        public string StackTypeId { get; private set; } = string.Empty;
+        public string? StackTypeId { get; private set; }
```

Comment:
this is required and has a custom type serializer. why is it nullable?

Reply:
as i have realized now, it has to be nullable or have a random default value. If it doesn't, then tests fail because it craps out when the component is added to an entity.